.backportrc.json

```
@@ -1,6 +1,6 @@
 {
   "upstream": "elastic/beats",
-  "branches": [ { "name": "7.x", "checked": true }, "7.12", "7.11" ],
+  "branches": [ { "name": "7.x", "checked": true }, "7.13", "7.12", "7.11" ],
```

Can probably get rid of 7.11 now that it's EOL.

```suggestion
  "branches": [ { "name": "7.x", "checked": true }, "7.13", "7.12" ],
```

---

Yes, please remove 7.11.